pilot/pkg/xds/v3/model.go

```diff
@@ -30,6 +30,9 @@ const (
 	SecretType                 = resource.SecretType
 	ExtensionConfigurationType = resource.ExtensionConfigType
 
+	// maistra xDS
+	TrustBundleType = "type.googleapis.com/maistra.security.v1.TrustBundleResponse"
```

Comment:
```suggestion
	TrustBundleType = resource.APITypePrefix + "maistra.security.v1.TrustBundleResponse"
```

Reply:
To be honest, I don't like string concatenation in cases like this, because it makes searching the code base more difficult when I work with config dump and want to quickly find usages of an API type.

Reply:
This prefix is used for all the other types and is extracted this way. Following that convention, we would potentially avoid issues if it will ever changes.

Reply:
Ok, you convinced me.

Reply:
![agree-startrek](https://media.giphy.com/media/KffdTQfewxdbKTGEJY/giphy.gif)